src/simple-content-moderation/plugin.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Simple content moderation plugin that prevents saving annotations containing hate speech
+ *
+ * This plugin monitors text entered into TextArea regions and checks for the word "hate"
+ * before allowing the annotation to be saved. If found, it shows an error message and
+ * prevents submission.
+ *
+ * The plugin uses Label Studio's beforeSaveAnnotation event which is triggered before
+ * an annotation is saved. Returning false from this event handler prevents the save
+ * operation from completing.
+ */
+
+let dismissed = false;
+
+LSI.on("beforeSaveAnnotation", (store, ann) => {
+	// text in TextArea is always an array
+	const obscene = ann.results.find(
+		(r) =>
+			r.type === "textarea" && r.value.text.some((t) => t.includes("hate")),
```

Comment:
Was that result of our linting? Don't believe it would format it like this
```suggestion
		(r) => r.type === "textarea" && r.value.text.some((t) => t.includes("hate")),
```

Reply:
Using the default biome rules (not the HS ones for now)